.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: Push Default
+on:
+  # push:
+  #   branches: [ master ]
+  push:
+    branches: [ auto-build ]
+    # paths: ['docker_config/dockerfiles/default-*']
```

Comment:
We should also trigger this on a release (see release.yml), in which case it should build and push these images with tag "latest".

Reply:
I updated the code so the workflow is run on PRs, push to masters, and release.